### PR TITLE
Add weekly post 2015-10-16

### PR DESCRIPTION
**Pull request by JungMinu**
Please review this post. :heart:,@nodejs/evangelism

### CODE REVIEW

File: weekly-updates/weekly-update.2015-10-16.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+slug: weekly-update-2015-10-16
+layout: blog-post.hbs
+---
+
+### Node.js News — October 16th
+Node.js v4.2.1 (LTS), v4.2.0 (LTS) are released
+
+### Node.js v4.2.1 (LTS) and v4.2.0 (LTS) Releases
+
+This week we have two releases: [Node.js v4.2.1 (LTS)](https://nodejs.org/en/blog/release/v4.2.1/) and [Node.js v4.2.0 (LTS)](https://nodejs.org/en/blog/release/v4.2.0/). complete changelog from previous releases can be found [on GitHub](https://github.com/nodejs/node/blob/master/CHANGELOG.md).
```
Comment (Trott) [inline weekly-updates/weekly-update.2015-10-16.md]: Capitalize first letter of `complete`.

File: weekly-updates/weekly-update.2015-10-16.md [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+
+### Notable changes
```
Comment (Trott) [inline weekly-updates/weekly-update.2015-10-16.md]: These are the notable changes for 4.2.1 only and not for 4.2.0.

File: weekly-updates/weekly-update.2015-10-16.md [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+
+* Includes fixes for two regressions
+  - Assertion error in WeakCallback  - see [#3329](https://github.com/nodejs/node/pull/3329)
+  - Undefined timeout regression - see [#3331](https://github.com/nodejs/node/pull/3331)
+
+### Known issues
```
Comment (Trott) [inline weekly-updates/weekly-update.2015-10-16.md]: Should we remove `Known Issues` in the weekly update? Seems like a lot of detail that most people probably skip over. If it's already in the linked announcements, that's probably enough.

File: weekly-updates/weekly-update.2015-10-16.md [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+* [A Hubot Plugin that Helps You to Choose Where to Drink Coffee](http://ditrospecta.com/javascript/plugin/hubot/2015/10/03/hubot-plugin-choose-place-to-drink-coffee.html), "Hubot is a very friendly and easy extensible bot written in Coffeescript"
+* [Building using Node-GYP with Visual Studio Express 2015 on Windows 10 Pro x64](http://www.serverpals.com/blog/building-using-node-gyp-with-visual-studio-express-2015-on-windows-10-pro-x64), "Setting up a development environment, using Node-GYP with Visual Studio Express 2015 on Windows 10 Pro x64"
+* [Linux and Node.js in Production using Daemontools and NDT](http://www.serverpals.com/blog/linux-nodejs-in-production-using-daemontools-ndt), "Putting the application to the test with Linux and Node.js in Production using Daemontools and NDT"
+* [Using Bower with BowerCDN for better web performance!](http://www.serverpals.com/blog/using-bower-with-bowercdn-for-better-web-performance), "BowerCDN is a service built on the bower API that installs and maintains permalinks to bower repository content"
+
```
Comment (Trott) [inline weekly-updates/weekly-update.2015-10-16.md:36]: I don't think we need all three serverpals.com links in one update. I would pick the best one or the most recent one.

File: weekly-updates/weekly-update.2015-10-16.md [hunk 5]
```diff
@@ -0,0 +1,54 @@
...
+If you have spotted or written something about Node.js, do come over to our [Evangelism team repo](https://github.com/nodejs/evangelism) and suggest it on the [Issues page](https://github.com/nodejs/evangelism/issues), specifically the Weekly Updates issue.
+
+### Upcoming Events
+
+* [JSConf CO](http://www.jsconf.co/), October 16th - 17th at Ruta N, Medellin, Colombia
+* [EmpireNode](http://2015.empirenode.org/), October 23rd at New York, US.
+* [NodeFest](http://nodefest.jp/2015/), November 7th at Stack Exchange HQ, Old St
```
Comment (Trott) [inline weekly-updates/weekly-update.2015-10-16.md]: The location is wrong for this entry. It should be `Tokyo, Japan`.

File: weekly-updates/weekly-update.2015-10-16.md [hunk 6]
```diff
@@ -0,0 +1,54 @@
...
+* [Node Girls London](https://nodegirls.typeform.com/to/atW4HR), November 7th at Stack Exchange HQ, Old St
```
Comment (Trott) [inline weekly-updates/weekly-update.2015-10-16.md]: I think `London, UK` is better for the location. The specific location is in the link. People don't need to know the building or street when scanning the events in our announcements. They just need the city and country. If interested, then click the link and get more specific details.

**Comment by Trott** [comment]
Thanks for putting this together!

**Comment by JungMinu** [comment]
@Trott Thanks for your comments :smile: 
I'll update soon based on your comments

**Comment by JungMinu** [comment]
@Trott I've updated :smile:

**Comment by Trott** [comment]
:+1:

**Comment by JungMinu** [comment]
I've created a pull request in `new.nodejs.org`: https://github.com/nodejs/new.nodejs.org/pull/248

**Comment by JungMinu** [comment]
This pull request is moved to https://github.com/nodejs/new.nodejs.org/pull/248

**Pull request closed without merging**